spec/support/admin_searchable_claim_shared_examples.rb

```diff
@@ -0,0 +1,115 @@
+RSpec.shared_examples "Admin Searchable Claim" do |policy|
+  subject(:search) { Claim::Search.new(query) }
+
+  let!(:other_claim) { create(:claim, :submitted) }
```

Comment:
This is not needed I think.

Reply:
I tried to follow the pattern of the existing tests. `other_claim` is there to show that the search isn't returning an unmatching claim. I could put it in a before block instead so it doesn't need a variable name?

Reply:
Ok, a `before` block should work then, it doesn't need a name.

Reply:
That's done